packages/pipeline/src/Counter.tsx

```diff
@@ -101,29 +34,38 @@ const getStatefulStyles = (state: StateForStyles) => ({
  */
 const Counter = forwardRef(
```

Comment:
I think @TheSonOfThomp would appreciate an `@internal` flag on this component while we're at it